<ui version="4.0" >
 <class>NewRosterItemDialogClass</class>
 <widget class="QDialog" name="NewRosterItemDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>New Roster Item</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="minimumSize" >
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>JID:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>edJID</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edJID" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="minimumSize" >
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Name:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>edName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edName" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewRosterItemDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>239</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel" >
     <x>243</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewRosterItemDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>166</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel" >
     <x>167</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
